udata_front/theme/gouvfr/assets/js/components/search/MenuSearch/MenuSearch.vue

```diff
@@ -61,15 +61,14 @@
 </template>
 
 <script lang="ts">
```

Comment:
Can you move the component to `script setup`syntax ? Now that the `setup` is only about 30 lines, it should be easy.